modular_skyrat/modules/liquids/code/liquid_systems/liquid_plumbers.dm

```diff
@@ -133,21 +132,20 @@
 	to_chat(user, span_notice("You turn [src] [turned_on ? "off" : "on"]."))
 	turned_on = !turned_on
 
-/obj/machinery/plumbing/liquid_output_pump/CtrlClick(mob/living/user)
+/obj/machinery/plumbing/liquid_output_pump/AltClick(mob/living/user)
 	if(anchored)
 		return ..()
 	if(!user.canUseTopic(src, BE_CLOSE, NO_DEXTERITY, FALSE, TRUE))
 		return
-	var/new_height = input(user, "Set new height regulation:\n([0]-[LIQUID_HEIGHT_CONSIDER_FULL_TILE]. Use 0 to disable the regulation)\nThe pump will only siphon if environment is below the regulation", "Liquid Pump") as num|null
-	if(QDELETED(src))
+	var/new_height = tgui_input_number(user, "Set new height regulation:\n([0]-[LIQUID_HEIGHT_CONSIDER_FULL_TILE]. Use 0 to disable the regulation)\nThe pump will only siphon if environment is below the regulation", "Liquid Pump", max_value = LIQUID_HEIGHT_CONSIDER_FULL_TILE, min_value = 0)
+	if(isnull(new_height) || QDELETED(src))
 		return
-	if(new_height)
-		height_regulator = sanitize_integer(new_height, 0, LIQUID_HEIGHT_CONSIDER_FULL_TILE, 0)
+	height_regulator = new_height
```

Comment:
It's good to keep the sanitization, because someone could decide to modify their client and allow themselves to input a much higher number, which could be a bad thing. (same for the other one above.